.github/workflows/Register.yml

```diff
@@ -0,0 +1,14 @@
+name: Register Package
+on:
```

Comment:
what's this?

Reply:
With this workflow, it registers a new version of the package just by specifying which part of the version to increase (major, minor, or patch).

More info:
https://github.com/julia-actions/RegisterAction

Reply:
But we have JuliaRegistrator setup everywhere. Any reason to not be using JuliaRegistrator?

Reply:
It works based on JuliaRegistrator. if we have it, there is no need to [manually increase the version](https://github.com/SciML/DiffEqFlux.jl/commit/092af896745e338d29c929b557e8728d4ef55327), or [trigger a release by commenting in a PR](https://github.com/SciML/DiffEqFlux.jl/issues/62).

Reply:
Ehh let's stick to JuliaRegistrator for now. It's quite a bit of an overhaul to train everyone on a new system if it doesn't give a tangible improvement. Also, updating all of the repos will be a nightmare.